docs/user/dql/troubleshooting.rst

```diff
@@ -0,0 +1,89 @@
+
+===============
+Troubleshooting
+===============
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Narrative
+=========
+
+SQL plugin is stateless for now so mostly the troubleshooting is mainly focused on why a single query fails.
```

Comment:
no need for `mostly` and `mainly` to both be in the sentence, it's redundant.

Reply:
Corrected now, thanks!